ci/launch_cluster.sh

```diff
@@ -13,13 +17,22 @@ CHANNEL="$1"
 VARIANT="$2"
 
 if [ "$VARIANT" == "open" ]; then
-  TEMPLATE="https://s3.amazonaws.com/downloads.dcos.io/dcos/${CHANNEL}/cloudformation/multi-master.cloudformation.json"
+  INSTALLER="https://downloads.dcos.io/dcos/${CHANNEL}/dcos_generate_config.sh"
+  EXTRA_DCOS_CONFIG=""
 else
-  TEMPLATE="https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise-aws-advanced/${CHANNEL}/${VARIANT}/cloudformation/ee.multi-master.cloudformation.json"
+  if [ -z "$DCOS_INSTALLER_TEMPLATE" ]; then
+      echo "Expecting DCOS_INSTALLER_TEMPLATE environment variable"
```

Comment:
Should we fail here?

Reply:
I was so sure I added an exit statement there :D Definitely